apps/extension/src/background/approvals/service.test.ts

```diff
@@ -94,7 +94,11 @@ describe("approvals service", () => {
     it("should throw an error when popupTabId is not present", async () => {
       jest.spyOn(service as any, "getPopupTabId").mockResolvedValue(undefined);
 
-      expect(service.approveSignature("signer", "data")).rejects.toBeDefined();
+      try {
```

Comment:
I think in these cases you could also just await or return:

```
await expect(service.approveSignature("signer", "data")).rejects.toBeDefined();
```

Reply:
Yeah I've noticed that a bit later :D

Reply:
I fixed that in another PR :)